# launcher_message.msg
# Part of imes_showroom
#
# Andreas Fregin (A.Fregin AT live.de)
# Leibniz Universitaet Hannover
#
# Version 1.0:		# 2013-02-23
Header header		# ROS std header
int32 demoNumber	# Number of the demo
time duration		# Duration that the demo shall run
string demoName		# Name of the Demo